src/MetricExporter.ts

```diff
@@ -184,14 +197,17 @@ export class DynatraceMetricExporter implements MetricExporter {
 		}
 
 		const payload = lines.join("\n");
+		this._sendRequest(payload, resultCallback, 0);
+	}
 
+	private _sendRequest(payload: string, resultCallback: (result: ExportResult) => void, retries: number) {
```

Comment:
The behavior here is a little confusing and I had to read it a couple times. Because it is not documented my first assumption was that this was the number of retries that should be done, but it is actually the number of retries already completed? I would prefer if this would count down instead of up. The first request on line 200 would be `this._sendRequest(payload, resultCallback, this._maxRetries)` and the recursive call would be `this._sendRequest(payload, resulltCallback, retries - 1)`